lib/irc/ClientPool.js

```diff
@@ -204,6 +204,20 @@ ClientPool.prototype._getNumberOfConnections = function(server) {
     return numConnectedNicks;
 };
 
+ClientPool.prototype.countTotalConnections = function() {
+    var count = 0;
+
+    Object.keys(this._virtualClients).forEach((domain) => {
```

Comment:
So, you want me to iterate the list of domain names in the map, converting each one into a server instance object (a process itself that involves a linear scan in the code linked above), just so I can pass the server instance into `_getNumberOfConnections` which will turn it back into a domain name to use as the key into the original map I iterated over in the first place?